gusto/solvers/linear_solvers.py

```diff
@@ -177,25 +177,20 @@ def __init__(self, equations, alpha=0.5, tau_values=None,
     def _setup_solver(self):
 
         equations = self.equations
+        cp = equations.parameters.cp
         dt = self.dt
         # Set relaxation parameters. If an alternative has not been given, set
         # to semi-implicit off-centering factor
-        beta_u_ = dt*self.tau_values.get("u", self.alpha)
-        beta_t_ = dt*self.tau_values.get("theta", self.alpha)
-        beta_r_ = dt*self.tau_values.get("rho", self.alpha)
+        beta_u = dt*self.tau_values.get("u", self.alpha)
+        beta_t = dt*self.tau_values.get("theta", self.alpha)
+        beta_r = dt*self.tau_values.get("rho", self.alpha)
+        beta_u_cp = Constant(beta_u * cp)
```

Comment:
Can we remove this variable and just have beta_u*cp in the code below?

Reply:
Should've known I couldn't get away with being lazy!